multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/security/UriAuthorizationFilter.java

```diff
@@ -14,4 +20,12 @@ public interface UriAuthorizationFilter {
      */
     boolean ensureUserIsAuthorized(HttpServletRequest request, HttpServletResponse response) throws IOException;
 
+    default void auditLogLoginAttempt(String username, String spaceGuid, String message) {
```

Comment:
This will be necessary to be extracted elsewhere if we refactor auditlogging logic to use beans instead static methods.